dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -1,15 +1,28 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 import "..\AppNotifications\AppNotifications.idl";
 
 namespace Microsoft.Windows.AppNotifications.Builder
 {
-    [contractversion(1)]
+    [contractversion(2)]
     apicontract AppNotificationBuilderContract {}
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationHeader
```

Comment:
Doc-comments are recommended for each runtimeclass

and desirable (and personally recommended) for everything else :-)